tests/Fixer/LanguageConstruct/DeclareBracesFixerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\LanguageConstruct;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\LanguageConstruct\DeclareBracesFixer
+ */
+final class DeclareBracesFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null): void
```

Comment:
Can we have types here and drop them from PHPDoc?

Also, `$input` is always `string`, never `null`.

Reply:
> Also, `$input` is always `string`, never `null`.

i would keep it like that for compatibility with `$this->doTest()`
i find it tedious to change test method prototype to change input type only because I added new test case. testX method is proxy to doTest, let's keep it's interface